scripts/tests/nextcloud.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+#shellcheck source=sources/functions/tests
+. /etc/swizzin/sources/functions/tests
+
+check_nginx "librespeed" || BAD="true"
```

Comment:
I don't think this is the correct nginx?

Reply:
You think correct, ha
```suggestion
check_nginx "nextcloud" || BAD="true"
```